Hello_Console_stud/Hello_Console_stud/Program.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Threading;
+
+namespace Hello_Console_stud
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a;
```

Comment:
what is mean `a`?

Reply:
Fixed in commit 592bca63f86e969b28bb074d314dd91abb86c457